src/components/Navbar.jsx

```
@@ -1,141 +1,255 @@
+  }
+
+  // Handle login navigation
+  navigateToLogin = () => {
+    window.location.href = '/login';
   }
 
   render() {
-    const { active, scroll } = this.state;
-    const { isDashboard } = this.props;
+    // Destructure state and props
+    const { active, scroll, isHomePage } = this.state;
```

ESLint

[error] 'isHomePage' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)